src/phoenix/datetime_utils.py

```diff
@@ -70,3 +70,16 @@ def floor_to_minute(dt: datetime) -> datetime:
 
 
 MINUTE_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:00%z"
+
+
+def time_range(
```

Comment:
this is not a standard time range, might name it a bit different, like `right_open_time_range` - it will read a bit cleaner at the callsite. otherwise it could be confusing why this is event needed.